src/components/Equipo/amigoSelect.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import {
+    Card,
+    CardHeader,
+    Avatar,
+    Box
+} from '@material-ui/core';
+import Rating from '@material-ui/lab/Rating';
+
+
+
+export class Amigo extends React.Component {
```

Comment:
Any reason for keeping class components here and using functional components in other files?